make_html.py

```diff
@@ -1,5 +1,6 @@
 from dataclasses import dataclass, field
 from enum import Enum
+import github
```

Comment:
This should be down by `jinja2`; the order is

* Builtin imports
* Third party imports

Reply:
done